src/sage/schemes/generic/morphism.py

```diff
@@ -100,8 +102,7 @@ def is_SchemeMorphism(f):
 
     OUTPUT:
 
-    Boolean. Return ``True`` if ``f`` is a scheme morphism or a point
```

Comment:
When touching any of these `is_...` functions, best to deprecate them. We are trying to get rid of all of them in favor of using `isinstance` (or categories) in the code. See #32414 for pointers

Reply:
I will do this in a separate PR